docs/secure-connections/sauce-connect/quickstart.md

```diff
@@ -42,7 +42,7 @@ These instructions assume that Sauce Connect Proxy is downloaded to your machine
   <TabItem value="Windows">
 
   ```bash
-  cd sc-4.7.1-win32/bin
+  cd C:\sc-4.7.1-win32\bin
```

Comment:
Hi @enriquegh @mdsauce @waggledans , can you confirm if this is how you'd navigate to your SC folder in a Windows home directory?